ros2param/test/test_verb_load.py

```diff
@@ -85,6 +85,12 @@
     '  ros__parameters:\n'
     '    str_param: Override\n'
 )
+INPUT_NS_NODE_OVERLAY_PARAMETER_FILE = (
+    f'{TEST_NAMESPACE}:\n'
+    f'  {TEST_NODE}:\n'
```

Comment:
This was never supported before, but it can be parsed by `rcl_param_yaml_parser`.